src/main/java/com/gmail/nossr50/events/skills/rupture/McMMOEntityDamageByRuptureEvent.java

```diff
@@ -0,0 +1,33 @@
+package com.gmail.nossr50.events.skills.rupture;
+
+import com.gmail.nossr50.datatypes.player.McMMOPlayer;
+import com.gmail.nossr50.events.fake.FakeEntityDamageByEntityEvent;
+import org.bukkit.entity.Entity;
+import org.bukkit.entity.Player;
+import org.bukkit.event.entity.EntityDamageEvent;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class McMMOEntityDamageByRuptureEvent extends FakeEntityDamageByEntityEvent {
```

Comment:
This is a big no-no, fake events are for things that don't actually happen. Don't use them for damage that will be real.

Reply:
ah, misunderstood what this was for then

Reply:
It's all good!

Reply:
You may want to have this extend SubSkillEvent (going off memory here) we should have some kind of "template" event for sub skills already in mcMMO

Reply:
I don't think I can do that? That'd be a class extending two classes

Reply:
You're about to remove the class it is extending, I was offering a suggestion of what it should be extending instead